packages/plugins/rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec

```diff
@@ -31,13 +31,18 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
 Requires: foreman-proxy >= %{foreman_proxy_min_version}
-Requires: ruby
-BuildRequires: ruby
+Requires: ruby >= 2.7
+Requires: ruby < 4
+BuildRequires: ruby >= 2.7
+BuildRequires: ruby < 4
 BuildRequires: rubygems-devel
 BuildArch: noarch
 Provides: foreman-proxy-plugin-%{plugin_name} = %{version}
 # end specfile generated dependencies
 
+Requires: /usr/bin/bunzip2
+Requires: /usr/bin/oscap
```

Comment:
I've debated whether I want file paths or package names here, and ended on paths, as that's what used in the code.

@adamruzicka @ekohl thoughst?

Reply:
I agree with paths because if the package ever gets changed, this will continue to work or show a clear error. Perhaps add a note that they're called at runtime?

Reply:
I am also fine with paths, if it makes things easier. But the disadvantage is this requires more metadata to be loaded from the repository which are going to be optional and not downloaded by default from EL10 onwards.